glossary/it/terms/marginal-cost/index.md

```diff
@@ -3,4 +3,4 @@ section: terms
 lang: it
 title: Costo marginale
 ---
-Si tratta del costo aggiuntivo sostenuto fornendo una singola copia di una risorsa, ad esempio, dati. Per i dati da aprire secondo la [Open Definition](/glossary/en/open-definition/), essi devono essere caricati non oltre il costo marginale. Qualora i dati sono disponibili per il download su Internet il costo marginale di solito è pari a zero. Ci può essere un piccolo costo marginale in casi eccezionali. Ad esempio le dimensioni dei dati necessitano di essere messi su un disco pubblicato. Confronta {cost-recovery}.
+Si tratta del costo aggiuntivo sostenuto fornendo una singola copia di una risorsa, ad esempio, dati. Per i dati da aprire secondo la [Open Definition](../open-definition/), essi devono essere caricati non oltre il costo marginale. Qualora i dati sono disponibili per il download su Internet il costo marginale di solito è pari a zero. Ci può essere un piccolo costo marginale in casi eccezionali. Ad esempio le dimensioni dei dati necessitano di essere messi su un disco pubblicato. Confronta {cost-recovery}.
```

Comment:
@morchickit I will delete `{}` and the separator `-` on it, but I guess it would be better to completely delete the last sentence `Confronta {cost-recovery}.`.